app/models/compute_resource.rb

```diff
@@ -184,7 +184,7 @@ def create_vm(args = {})
   end
 
   def destroy_vm(uuid)
-    find_vm_by_uuid(uuid).destroy
+    find_vm_by_uuid(uuid).destroy :force => true
```

Comment:
@agx If this is just for VMWare - mind to move the call to `vmware.rb`? It'd be as simple as rewriting the method in that file, you can see other methods like `new_vm` etc... have been overridden there already.

The reason is just that we cannot ensure all fog compute resource providers accept this option (or other options at all). Azure for example does not accept any argument on .destroy calls